src/components/codeblocks/CodeblockTree.svelte

```diff
@@ -30,19 +36,26 @@
 	export const update = () => {
 		const max_depth = options.depth[1] ?? DEFAULT_MAX_DEPTH;
 
+		const fields =
+			options.fields ?? plugin.settings.edge_fields.map((f) => f.label);
+
+		const source_path = file_path
+			? file_path
+			: $active_file_store
+				? $active_file_store.path
+				: "";
+
 		const traversal_options = new TraversalOptions(
-			[file_path],
-			options.fields ?? [],
```

Comment:
Question here. Previously the fallback was `[]`. Does your traversal treat an empty fields list as _all_ fields allowed? Cause mine needed you to explicitly pass in the full list of fields, if that's what you wanted

Reply:
`undefined` is all fields, `[]` is none

Reply:
Ok call, I'll update my change to use `undefined` when necessary